<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Associazione</class>
 <widget class="QMainWindow" name="Associazione">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Associazione studente-professore</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Downloads/textdocuments_102739.png</normaloff>../../Downloads/textdocuments_102739.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDialogButtonBox" name="buttonBoxAssociazione">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>750</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Yes</set>
    </property>
    <property name="centerButtons">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxStudente">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>301</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Matteo Chiocci</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Edoardo Panfili</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Samanta Satiri</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Maria Teresa Rughi</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="titoloPagina">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light SemiCondensed</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Associazione studente-professore</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxInsegnante">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>301</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>David Stocchi</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Marco Meletti</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Valeria Giovannini</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Matteo Picciolini</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelAssociazione">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>730</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Creo questa associazione?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelInsegnante">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Inserire il nominativo dell'insegnante</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStudente">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Inserire il nominativo dello studente</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOreRimanenti">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ore rimanenti dell'insegnante:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOreDaAssegnar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>650</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ore da assegnare:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOre">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>650</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelOre">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>310</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>18</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelInseriemntoVeloceStudente">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Inserimento veloce </string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInserimentoVeloceInsegnante">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Inserimento veloce </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditStudente">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>301</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditInsegnanti">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>301</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStudente">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>560</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>cerca</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInsegnante">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>260</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>cerca</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
